values.yaml

```diff
@@ -0,0 +1,35 @@
+redis:
+  enabled: true
+  auth:
+    enabled: false
+  architecture: standalone
+  master:
+    persistence:
+      enabled: true
+      size: 1Gi  # Adjust the size as needed
+
+onetimesecret:
+  image: 
+    registry: docker.io
+    repository: onetimesecret/onetimesecret
+   #tag:latest  ##This will override the .Chart.AppVersion
+  service:
+    type: ClusterIP
+  env:
+    COLONEL: "admin@example.com"
+    SSL: "false"  # Ensure this is a string
+    SMTP_HOST: "smtp.yourdomain.com" # Update your smtp configuration details
```

Comment:
For consistency i would use "example.com" instead of "yourdomain.com". I believe example.com is owned by IANA while yourdomain.com is owned by an individual/org.

Reply:
ohh..you are right. I have replaced it in latest commit